src/status_im/ui/screens/contacts/events.cljs

```diff
@@ -277,7 +277,10 @@
     (let [contacts-list (map #(vector (:whisper-identity %) %) all-contacts)
           contacts (into {} contacts-list)]
       {:db         (assoc db :contacts/contacts contacts)
-       :dispatch-n (mapv (fn [_ contact] [:watch-contact contact]) contacts)})))
+       ;; TODO (yenda) this mapv was dispatching useless events, fixed but is it necessary if
```

Comment:
We probably can leave this comment in PR, not sure that we need comments for code that doesn't exist anymore :)

Reply:
I want to leave it for further investigation on discovery unless you are sure it is really useless ? I must have been put there for some reason

Reply:
then i think better to leave commented code

Reply:
oh its there. Then ok

Reply:
previously the parameters of the anonymous function were not destructured properly when user logged in a dozen of these watch-contact events were dispatch with nil as parameter.